io.catenax.digital_product_passport/1.0.0/DigitalProductPassport.ttl

```diff
@@ -0,0 +1,846 @@
+:Either a bamm-c:Either;
+    bamm:preferredName "Either"@en;
+    bamm:description "Characteristic with two possible  values."@en;
+    bamm-c:left :RangeCharacteristic;
+    bamm-c:right :ConcentrationCharacteristic.
+:materialName a bamm:Property;
+    bamm:preferredName "Material Name"@en;
+    bamm:description "The material name which is part of the product."@en;
+    bamm:characteristic bamm-c:Text;
+    bamm:exampleValue "Aluminium".
+:renewable a bamm:Property;
+    bamm:preferredName "Renewable"@en;
+    bamm:description "Flag if the material is made of natural resources that can be replenished. Renewable resources are those that can be reproduced by physical, chemical, or mechanical processes. These are the kind of resources that can be regenerated throughout time. Forest wood, for example, can be grown through reforestation."@en;
+    bamm:characteristic bamm-c:Boolean;
+    bamm:exampleValue "true"^^xsd:boolean.
+:percentage a bamm:Property;
+    bamm:preferredName "Percentage"@en;
+    bamm:description "Percentage of the material."@en;
+    bamm:characteristic :Percent;
+    bamm:exampleValue "5.0"^^xsd:float.
+:Percent a bamm-c:Measurement;
+    bamm:preferredName "Percent"@en;
+    bamm:dataType xsd:float;
+    bamm-c:unit unit:percent.
+:lifeUnit a bamm:Property;
+    bamm:preferredName "Life Unit"@en;
+    bamm:description "The unit of the expected lifetime of a product from the unit catalog."@en;
+    bamm:see <https://eclipse-esmf.github.io/samm-specification/snapshot/appendix/unitcatalog.html>;
+    bamm:characteristic :LifespanEnumeration;
+    bamm:exampleValue "unit:month".
+:LifespanEnumeration a bamm-c:Enumeration;
+    bamm:preferredName "Lifespan Enumeration"@en;
+    bamm:see <https://eclipse-esmf.github.io/samm-specification/snapshot/appendix/unitcatalog.html>;
+    bamm:dataType xsd:string;
+    bamm-c:values ("unit:day" "unit:month" "unit:cycle" "unit:year" "unit:kilometre" "unit:runningOrOperatingHour").
+:lifeValue a bamm:Property;
+    bamm:preferredName "Life Value"@en;
+    bamm:description "The value of the expected lifetime of a product."@en;
+    bamm:characteristic :LifeValueCharacteristic;
+    bamm:exampleValue "36"^^xsd:integer.
+:LifeValueCharacteristic a bamm-c:Quantifiable;
+    bamm:preferredName "Life Value Characteristic"@en;
+    bamm:dataType xsd:integer.
+:ownerId a bamm:Property;
+    bamm:preferredName "Owner Id"@en;
+    bamm:description "The identification of the owner of the product and the passport."@en;
+    bamm:characteristic :CompanyIdentification;
+    bamm:exampleValue "BPNL1234567890ZZ".
+:eori a bamm:Property;
+    bamm:preferredName "EORI"@en;
+    bamm:description "Any economic operator established in the customs territory of the Union needs, for customs purposes, an EORI number. EORI stands for economic operators registration and identification number."@en;
+    bamm:characteristic :EoriTrait;
+    bamm:exampleValue "DE 123 456 789 12345".
```

Comment:
have you checked that the example validates against the defined regex constraint?

Reply:
fixed example and regex